docs/classes.rst

```diff
@@ -488,6 +488,24 @@ The entries defined by the enumeration type are exposed in the ``__members__`` p
     >>> Pet.Kind.__members__
     {'Dog': Kind.Dog, 'Cat': Kind.Cat}
 
+The ``name`` property returns the name of the enum value as a unicode string.
+
+.. note::
+
+    It is also possible to use ``py::str(enum)``, however these accomplish different
```

Comment:
Typo: that should just be `str(enum)` (i.e. no `py::`), since this is a Python reference.